client/src/cordova/plugin/apple/src/OutlinePlugin.swift

```diff
@@ -154,10 +154,12 @@ class OutlinePlugin: CDVPlugin {
     DDLogInfo("Invoking Method \(methodName) with input \(input)")
     Task {
       guard let result = OutlineInvokeMethod(methodName, input) else {
+        DDLogInfo("InvokeMethod \(methodName) got nil result")
         return self.sendError("unexpected invoke error", callbackId: command.callbackId)
       }
       if result.error != nil {
         let errorJson = marshalErrorJson(error: OutlineError.platformError(result.error!))
+        DDLogInfo("InvokeMethod \(methodName) failed with error \(errorJson)")
```

Comment:
Also `Warn`?

```suggestion
        DDLogWarn("InvokeMethod \(methodName) failed with error \(errorJson)")
```

Reply:
I don't want to leak errors to Sentry. Changed to debug.